cycles-ledger/tests/tests.rs

```diff
@@ -149,35 +152,67 @@ fn install_fake_cmc(env: &StateMachine) {
     );
 }
 
+/** Create an ICRC-1 Account from two numbers by using their big-endian representation */
+pub fn account(owner: u64, subaccount: Option<u64>) -> Account {
+    Account {
+        owner: Principal::from_slice(owner.to_be_bytes().as_slice()),
+        subaccount: subaccount
+            .map(|subaccount| subaccount.to_be_bytes().as_slice().try_into().unwrap()),
+    }
+}
+
 #[test]
 fn test_deposit_flow() {
     let env = &new_state_machine();
     let ledger_id = install_ledger(env);
     let depositor_id = install_depositor(env, ledger_id);
-    let user = Account {
-        owner: Principal::from_slice(&[0]),
-        subaccount: None,
-    };
+    let user = account(0, None);
 
-    // Check that the total supply is 0
+    // Check that the total supply is 0.
     assert_eq!(total_supply(env, ledger_id), 0u128);
 
     // Check that the user doesn't have any tokens before the first deposit.
     assert_eq!(balance_of(env, ledger_id, user), 0u128);
 
     // Make the first deposit to the user and check the result.
-    let deposit_res = deposit(env, depositor_id, user, 1_000_000_000);
+    let deposit_res = deposit(env, depositor_id, user, 1_000_000_000, None);
     assert_eq!(deposit_res.block_index, Nat::from(0_u128));
     assert_eq!(deposit_res.balance, Nat::from(1_000_000_000_u128));
 
-    // Check that the right amount of tokens have been minted
+    // Check that the right amount of tokens have been minted.
     assert_eq!(total_supply(env, ledger_id), 1_000_000_000);
 
     // Check that the user has the right balance.
     assert_eq!(balance_of(env, ledger_id, user), 1_000_000_000);
 
+    // Check that the block created is correct
```

Comment:
```suggestion
    // Check that the block created is correct:
```

Reply:
Do we want to change all the comments to include `:` at the end? I can do them in this MR if you want.

Reply:
No, only if the comment is followed by a list of items that all contribute to correctness.
So, the two fixes that I suggested should be sufficient.